src/visualizer.js

```diff
@@ -484,6 +477,36 @@ export default class Visualizer {
         if (preset.shapes[i].baseVals.enabled !== 0) {
           preset.shapes[i].init_eqs = mod.exports[`shapes_${i}_init_eqs`];
           preset.shapes[i].frame_eqs = mod.exports[`shapes_${i}_frame_eqs`];
+
+          const resetMod = await Visualizer.makeResetModule(
+            wasmVarPools[`shapePerFrame${i}`],
```

Comment:
I think you can do this once, rather than once per shape equation, since the globals are the same in each one. That will save us `n - 1` Wasm modules.

Reply:
So we can definitely do it in one module, but we will still need for separate functions because each shape has its own varpool with globals of the same name: https://github.com/jberg/butterchurn/blob/23c72aebdb7e22dec51d45ae5b8580c9f2a09072/src/visualizer.js#L413